src/capellambse_context_diagrams/collectors/default.py

```diff
@@ -47,7 +49,7 @@ def collector(
             visited_comps.add(current.uuid)
 
             inc, out = _generic.port_collector(current, diagram.type)
-            for port in chain(inc.values(), out.values()):
+            for port in itertools.chain(inc.values(), out.values()):
```

Comment:
There's some unrelated changes mixed into this commit as well, like whitespace / blank line changes in `builders/default.py` and this `from itertools import chain -> import itertools`. I know it's still marked WIP, just leaving this here so we don't forget about it later :)

Reply:
That's okay for me.